libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -410,6 +410,16 @@ AP_Mount::AP_Mount()
 	AP_Param::setup_object_defaults(this, var_info);
 }
 
+bool AP_Mount::check_primary() const
```

Comment:
Not a big deal but to make it easier for developers to find the code they're looking for I normally try to keep the implementations in the same order as the definitions.  So in this case I'd move these new private method implementations much lower in the .cpp file.  For example they could appear just before handle_gimbal_report.